docs/querying/sql.md

```diff
@@ -47,11 +47,11 @@ Druid SQL supports SELECT queries with the following structure:
 ```
 [ EXPLAIN PLAN FOR ]
 [ WITH tableName [ ( column1, column2, ... ) ] AS ( query ) ]
-SELECT [ ALL | DISTINCT ] { * | exprs }
+SELECT [ ALL | DISTINCT ] { * | exprs } [ FILTER ( WHERE expr ) ]
```

Comment:
I don't think this belongs in the query syntax description at all. It should be a section in the aggregation functions as it can only apply there. Have a look at how it is implemented on [Calcite's reference page](https://calcite.apache.org/docs/reference.html) (search for `FILTER (`)